presto-docs/src/main/sphinx/release/release-338.rst

```diff
@@ -0,0 +1,49 @@
+=========================
+Release 338 (XX Jul 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect results when joining tables on a masked column. (:issue:`4251`)
+* Fix planning failure when multiple columns have a mask. (:issue:`4322`)
+* Fix incorrect comparison for ``TIMESTAMP WITH TIME ZONE`` values with precision larger than 3. (:issue:`4305`)
+* Fix query failure when using ``VALUES`` with a ``real`` or ``double`` ``NaN`` value. (:issue:`4119`)
+* Fix query failure when joining tables on a ``real`` or ``double`` column and one of the joined tables
+  contains ``NaN`` value. (:issue:`4272`)
+* Fix incorrect query results when Parquet file has no min/max statistics for an integer column. (:issue:`4200`)
```

Comment:
is "integer" the right term to describe a smallint/integer/bigint types?

Reply:
Maybe "integral" would be more appropriate.